root/scripts/talker/coach.txt

```diff
@@ -3352,6 +3352,29 @@ Rule EllisStoryReac01Coach
 	Response EllisStoryReac01Coach
 }
 
+Rule EllisStoryInterruptC13M2Coach
```

Comment:
Again, these rules aren't necessary because the other survivors do not have a map specific EllisInterrupt rule for these maps. The generic version will still be used instead.

Reply:
Nick does